client/common/svg.download.js

```diff
@@ -1,5 +1,7 @@
 import { select } from 'd3-selection'
 import { to_svg } from '../src/client'
+import { jsPDF } from 'jspdf'
+import 'svg2pdf.js'
```

Comment:
is `svg2pdf.js` required? there are no functions imported or called from it

Reply:
If I removed it I get this error
`svg.download.js:123 Uncaught (in promise) TypeError: doc.svg is not a function`